spec/unit/resource_spec.rb

```diff
@@ -283,7 +283,7 @@ def expect_lookup(key, options = {})
     let(:scope)    { Puppet::Parser::Scope.new(compiler) }
 
     def ast_leaf(value)
-      Puppet::Parser::AST::Leaf.new({:value => value})
+      Puppet::Parser::AST::Leaf.new(**{:value => value})
```

Comment:
Can avoid the "create hash to double splat" conversion by passing the keyword argument directly `Leaf.new(value: value)`